test/jdk/java/awt/Cursor/CursorDragTest/ListDragCursor.java

```diff
@@ -78,7 +76,61 @@ public static Frame createUI() {
         panel.add(list);
 
         frame.add(panel);
-        frame.setBounds(300, 100, 300, 150);
-        return frame;
+        frame.setSize(300, 150);
+        frame.setLocation(450, 400);
+        frame.setVisible(true);
+    }
+
+    static void createInstructionsFrame() {
+        String INSTRUCTIONS = """
+                1. Move mouse to the TextArea.
+                2. Press the left mouse button.
+                3. Drag mouse to the list.
+                4. Release the left mouse button.
+
+                The mouse cursor should appear as an I-beam cursor
+                and should stay the same while dragging across the
+                components. Once you reach the list, release the
+                left mouse button. Immediately after, the cursor
+                should change to a Hand cursor. If true, this test
+                passes.
+
+                The test fails if the cursor updates while dragging
+                over the components before releasing the left
+                mouse button.
+                """;
+
+        TextArea textArea = new TextArea(INSTRUCTIONS,
+                15, 60, TextArea.SCROLLBARS_NONE);
+        Button passBtn = new Button("PASS");
+        Button failBtn = new Button("FAIL");
+        Panel btnPanel = new Panel(new GridBagLayout());
+        Panel panel = new Panel(new GridBagLayout());
+        instructionsFrame = new Frame("Test Instructions");
+        passBtn.setMaximumSize(new Dimension(100, 30));
+        failBtn.setMaximumSize(new Dimension(100, 30));
+        btnPanel.add(passBtn);
+        btnPanel.add(failBtn);
+        passBtn.addActionListener(e -> disposeFrames());
+        failBtn.addActionListener(e -> {
+            disposeFrames();
+            throw new RuntimeException("Test Failed");
+        });
+        panel.add(textArea);
+        panel.add(btnPanel);
+        instructionsFrame.add(panel);
+        instructionsFrame.pack();
+        instructionsFrame.setLocation(300, 100);
+        instructionsFrame.setVisible(true);
```

Comment:
Rearranging the lines provides better clarity. mainPanel layout can be set to BorderLayout for better placement of the instruction panel and button panel.

```suggestion
        instructionsFrame = new Frame("Test Instructions");
        Panel mainPanel = new Panel(new BorderLayout());
        TextArea textArea = new TextArea(instructions, 15, 60,
                                         TextArea.SCROLLBARS_NONE);

        Panel btnPanel = new Panel();
        Button passBtn = new Button("PASS");
        Button failBtn = new Button("FAIL");
        btnPanel.add(passBtn);
        btnPanel.add(failBtn);

        passBtn.addActionListener(e -> disposeFrames());
        failBtn.addActionListener(e -> {
            disposeFrames();
            throw new RuntimeException("Test Failed");
        });

        mainPanel.add(textArea, BorderLayout.CENTER);
        mainPanel.add(btnPanel, BorderLayout.SOUTH);

        instructionsFrame.add(mainPanel);
        instructionsFrame.pack();
        instructionsFrame.setLocation(300, 100);
        instructionsFrame.setVisible(true);
```

Reply:
Thanks for the suggestion. Updated the test with the list of changes.